app/models/company.rb

```diff
@@ -55,7 +55,36 @@ def latest_cp_assessment
     cp_assessments.order(:assessment_date).last
   end
 
-  def sector_benchmarks
-    sector.cp_benchmarks
+  def latest_sector_benchmarks
+    sector.latest_released_benchmarks
+  end
+
+  # Returns sector CP benchmarks:
+  # - if company has assessment      - from the last date before latest CP::Assessment date
+  # - if company has no assessments  - from latest benchmarks
+  # - if company has old assessments - from latest benchmarks
```

Comment:
what does it mean "old assessments" here?

Reply:
old = "from before the oldest benchmark", I'll update it.

Reply:
so then the oldest benchmark should be used, right?

Reply:
ahh, no, then I don't know what should be used :)